tests/unit/EntityMenu.spec.ts

```diff
@@ -528,9 +535,12 @@ describe('Entity Menu - Request AGM Location Change click tests', () => {
       propsData: { businessId: 'BC1234567' }
     })
 
+    vm = wrapper.vm
+    vi.spyOn(vm, 'enableAgmLocationChg', 'get').mockReturnValue(true)
+
     await wrapper.find('.menu-btn').trigger('click')
 
-    expect(wrapper.find('#agm-loc-chg-list-item').exists()).toBe(true)
+    expect(wrapper.find('#agm-loc-chg-list-item').exists()).toEqual(true)
```

Comment:
"toBe" is more accurate.

Reply:
My reasoning:

"toBe" is like `===`.
"toEqual" is like `==`.

In this case, since you're comparing a primitive (boolean value), they're equivalent. But for comparing objects/arrays, we sometimes have to use "toEqual".